.github/workflows/dockerhub-mirror.yml

```diff
@@ -2,7 +2,7 @@
 name: Mirror Dockerhub
 env:
   REGISTRY: ghcr.io
-  CONTAINER_REPO: ${{ github.repository }}
+  CONTAINER_REPO: ${{ github.repository_owner }}
```

Comment:
You can add a comment giving an example of full image name for busybox.

Reply:
Added a top comment documenting the workflow and indicating where images are pushed to.